src/js/metadata.js

```diff
@@ -293,7 +293,7 @@ export default {
   'Iteration': ['cascade', 'iterate', 'repeat', 'stack', 'steps'],
   'Java': ['coffee', 'code', 'coding', 'cup'],
   'Js': ['javascript', 'logo', 'react', 'reactjs'],
-  'Keyboard': ['accessibility', 'tab'],
+  'Keyboard': ['accessibility', 'keyboard', 'tab', 'technology'],
```

Comment:
```suggestion
  'Keyboard': ['accessibility', 'tab', 'technology'],
```
The name of the icon is used as a default search option, so it doesn't need to be added to the metadata.